db/migrate/20230227152913_change_reminder_emails_to_polymorphic_association.rb

```diff
@@ -0,0 +1,15 @@
+class ChangeReminderEmailsToPolymorphicAssociation < ActiveRecord::Migration[
+  7.0
+]
+  def change
+    change_table :reminder_emails, bulk: true do |t|
+      t.remove_foreign_key :further_information_requests
+      t.remove_index [:further_information_request_id]
+      t.rename :further_information_request_id, :requestable_id
+      t.string :requestable_type,
+               null: false,
+               default: "FurtherInformationRequest"
```

Comment:
Does the migration work if we then immediately after this line change the default to `nil`? I don't think it matters much, but feels a bit odd to leave the default here I think.

Reply:
Good idea, I'll give it a go.

Reply:
It works if the `change_column_default` method is called outside of the `change_table` block ie. we issue 2 `ALTER` statements to ensure the default is applied to existing records then reset to an empty string. I've amended the commit.